ts/features/bonus/cgn/screens/merchants/CgnMerchantsListByCategory.tsx

```diff
@@ -31,25 +38,33 @@ import {
   cgnOfflineMerchantsSelector,
   cgnOnlineMerchantsSelector
 } from "../../store/reducers/merchants";
-import { CATEGORY_GRADIENT_ANGLE, getCategorySpecs } from "../../utils/filters";
+import { getCategorySpecs } from "../../utils/filters";
 import { mixAndSortMerchants } from "../../utils/merchants";
 import { ProductCategoryEnum } from "../../../../../../definitions/cgn/merchants/ProductCategory";
+import { useHeaderSecondLevel } from "../../../../../hooks/useHeaderSecondLevel";
+import FocusAwareStatusBar from "../../../../../components/ui/FocusAwareStatusBar";
+import CgnMerchantsListView from "../../components/merchants/CgnMerchantsListView";
 
 export type CgnMerchantListByCategoryScreenNavigationParams = Readonly<{
   category: ProductCategoryEnum;
 }>;
 
-const styles = StyleSheet.create({
-  listContainer: {
-    paddingTop: 5,
-    borderTopLeftRadius: 25,
-    borderTopRightRadius: 25,
-    backgroundColor: IOColors.white,
-    top: -20
-  }
-});
-
 const CgnMerchantsListByCategory = () => {
+  const [titleHeight, setTitleHeight] = React.useState(0);
+  const translationY = useSharedValue(0);
+
+  const getTitleHeight = (event: LayoutChangeEvent) => {
+    const { height } = event.nativeEvent.layout;
+    if (titleHeight === 0) {
+      setTitleHeight(height);
```

Comment:
```suggestion
      setTitleHeight(height - insets.top - IOVisualCostants.headerHeight);
```
What do you think about setting the size of the title while also considering the size of the safe area and the header? By doing so, the snap point and opacity animation results accurate as soon as the header touches the list as shown below.

|Before|After|
|-|-|
|<video src="https://github.com/pagopa/io-app/assets/34343582/9d77cbe1-04c8-4b51-aa5e-91cdf444af0f" />|<video src="https://github.com/pagopa/io-app/assets/34343582/c021f038-852d-4eac-b477-17fd06a465a1" />